test/integration/queue/dao/FashionKnex-test.js

```diff
@@ -0,0 +1,125 @@
+require('require-self-ref')
```

Comment:
Why is this test under `test/integration/queue/dao/`? Maybe put it under `test/integration/dao/` instead? (same applies for other integration tests)